pdgstaging/TileStager.py

```diff
@@ -421,8 +418,18 @@ def save_tiles(self, gdf=None):
                     warnings.simplefilter('ignore', FutureWarning)
                     data.to_file(tile_path, mode=mode)
 
+                # convert each tile from string format to morecantile format 
+                # so it can be added to summary
+                # first create series of tiles in str format
+                tiles_str = data[self.props['tile']].copy()
+
+                tiles_morecantile = []
+                for tile in tiles_str:
+                    tile_morecantile = self.tiles.tile_from_str(tile)
+                    tiles_morecantile.append(tile_morecantile)                
```

Comment:
@julietcohen - there is nothing wrong with how you wrote your loop here, feel free to reject this suggestion, I just wanted to show you a syntax for combining these four lines into one! It's called [list comprehension](https://www.w3schools.com/python/python_lists_comprehension.asp) if you have not heard of it before:

```suggestion
                tiles_morecantile = [self.tiles.tile_from_str(tile) for tile in tiles_str]
```